docs/core/compatibility/serialization/9.0/jsondocument-props.md

```diff
@@ -1,12 +1,12 @@
 ---
-title: "Nullable JsonDocument properties deserialize to JsonValueKind.Null"
+title: "Breaking change: Nullable JsonDocument properties deserialize to JsonValueKind.Null"
 description: Learn about the breaking change in serialization in .NET 9 where nullable JsonDocument values deserialize to JsonValueKind.Null instead of null.
 ms.date: 12/5/2024
 ai-usage: ai-assisted
 ms.custom: https://github.com/dotnet/docs/issues/43869
 ---
 
-# Nullable JsonDocument properties deserialize to JsonValueKind.Null
+# Breaking change: Nullable JsonDocument properties deserialize to JsonValueKind.Null
```

Comment:
Did you mean to change the H1 too?

Reply:
I did, but I'm realizing now I was wrong.